server/conf/messages

```diff
@@ -328,7 +328,7 @@ content.confirmed=Thank you for applying to {0}.  Your application has been rece
 title.createAnAccount=Create an account or sign in
 
 # A message urging users to create an account to save their data.
-content.pleaseCreateAccount=If you sign into a city account now, your information will be saved and you''ll be able to return at any time to complete future applications.  If you do not already have an account, the login page will allow you to register.  You won''t lose any data you''ve already entered.
+content.pleaseCreateAccount=Log in to your City of Seattle account or create  an account to have your information saved. You'll be able to return at any time to complete future applications. If you do not already have an account, the login page will allow you to sign up and you won''t lose any data you?ve already entered.
```

Comment:
question(blocking): Is it intentional that this text is Seattle specific? I'd have expected a text that inserts the civic entity name here.

Reply:
Also, should it be "you've already entered" instead of "you?ve already entered"?